go.mod

```diff
@@ -3,8 +3,15 @@ module github.com/linuxboot/fiano
 go 1.16
 
 require (
+	github.com/dustin/go-humanize v1.0.0
+	github.com/fatih/camelcase v1.0.0
+	github.com/fatih/structtag v1.2.0 // indirect
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.7.0
```

Comment:
I'm not sure about testify.
Fiano uses "custom" checks in tests

Reply:
Ouh. OK. I can use standard `testing` package instead. Will fix.

Reply:
I'm fine with this. If anyone complains, we can still move on. :)